docs/Tutorials/TLS.md

```diff
@@ -3,53 +3,21 @@ description: TLS
 
 # TLS
 
-Orion supports the Transport Layer Security (TLS) protocol to enable secure communications. TLS is used between Orion nodes only. TLS is not used between Besu and Orion nodes.
+Orion supports the Transport Layer Security (TLS) protocol to enable secure
+communications between clients (for example Besu) and Orion, and
+between Orion nodes. 
 
-Enable TLS by setting the `tls` property to `strict` in the [Orion configuration file](../Reference/Configuration-File.md).  
+Enable TLS and set TLS properties in the [Orion configuration file](../Reference/Configuration-File.md):
 
-## Generating certificates using Orion
+* For client TLS, set the `clientconnectiontls` property to
```

Comment:
A link to the `clientconnectiontls` heading might be useful

Reply:
Done